wiki/Tournament_Drawings/zh.md

```diff
@@ -0,0 +1,88 @@
+<!-- lazer在持续更新，所以请记得更新！ 最后更新日期2021年1月 -->
+
+<!--  -->
+
+# 比赛抽签
+
+有些锦标赛可能包括小组赛阶段。在 [osu!lazer比赛端](/wiki/osu!lazer/tournament_client) 中，可以利用比赛抽签系统对小组赛的分组仪式进行现场直播。
```

Comment:
this line is different from the original, for ref:

```md
The tournament drawings screen is used to livestream the sorting of teams into groups that will compete in the group stages for a tournament. It is only available in the [osu!lazer](https://github.com/ppy/osu/releases) client.
```

Reply:
The purpose of the Chinese translation group is to translate the original text and supplement the text so that Chinese players can understand the content of the article. It is normal that most articles differ greatly from the original text. Updates to the English text will be given to other members.
中文翻译组的目的是将原文进行翻译并补充让中文玩家能够理解该文章内容，大部分文章与原文有较大出入是正常的。英文原文的更新将交给其他成员。

Reply:
Note that "the Chinese translation group" here is not @osu-translate-zh , we never said that. And it's unusual that "most articles differ greatly from the original text", that's totally stray from osu-wiki's principle, content parity[1]
此处的中文翻译组并非 @osu-translate-zh ，我们没有说过这事。“大部分文章与原文有较大出入”是异常的，这完全违背了 osu-wiki 的原则，即内容对等[1]

Why not do it yourself?
自己动手写一下？

[1] https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#content-parity / https://osu.ppy.sh/wiki/zh/Article_styling_criteria#%E5%86%85%E5%AE%B9%E5%AF%B9%E7%AD%89